qrify/qrify

```diff
@@ -108,23 +125,35 @@ Usage: qrify [stringtoturnintoqrcode]
     -m  Enable multiline support (feature not working yet)
     -h  Show the help
     -v  Get the tool version
+		-f  Store the QR code as a PNG file
 Examples:
     qrify this is a test string
     qrify -m two\\\\nlines
     qrify github.com (no http:// or https://)
+		qrify -f fileoutput.png google.com
 EOF
 }
 
 getConfiguredClient || exit 1
 
 
-while getopts "m:hvu*:" option
+while getopts "m:hvuf:*:" option
 do
   case "${option}" in
     v) echo "Version ${currentVersion}" && exit 0 ;;
     u) checkInternet && update && exit 0 || exit 1 ;;
     h) usage && exit 0 ;;
     m) multiline="1" && echo "Error this is not a supported feature yet" && exit 1 ;;
+		f) 
```

Comment:
The option here should only do ```fileName=$OPTARG``` this will allow for checking if the user has entered any other flags (even though the -m option isn't currently working if it ever does this will have to change anyways) how would you handle ```qrify -m -f imagename "a two \n line qrcode```